packages/xml-views-validation/src/validators/elements/use-of-depracated-aggregation.ts

```diff
@@ -0,0 +1,40 @@
+import { XMLElement } from "@xml-tools/ast";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import { getUI5AggregationByXMLElement } from "@ui5-language-assistant/logic-utils";
+import { UseOfDeprecatedAggregationIssue } from "../../../api";
+import {
+  buildDeprecatedIssueMessage,
+  DeprecatedUI5Symbol,
+} from "../../utils/deprecated-message-builder";
+
+export function validateUseOfDeprecatedAggregation(
```

Comment:
This may be very very similar to deprecatedClass.
Perhaps they can/should be joined?